### ISSUE TITLE
TypeError: r is not async iterable

### ISSUE THREAD

**Issue by JohnRSim** [lukeed/fetch-event-stream#3]
"fetch-event-stream": "^0.1.3",

![image](https://github.com/lukeed/fetch-event-stream/assets/364208/1eab4485-c7b0-44f5-9616-edd5ebcc7833)

Getting this error in chrome and edge windows latest stable - (v123.0.2420.65) 

On chrome canary 125.0.6384.0 works fine no errors.

```
	const response = await fetch(`${PUBLIC_BASE_PATH}/api/ai`, { 
		method: 'POST',
                headers:{
                	'fb_api_key':PUBLIC_AI_KEY
                },
                body: JSON.stringify({
                    message:msg,
                    userId:$page.data.session.user.email,
                    aiType:PUBLIC_AI_TYPE,
                    conversationId:chatID || createConversationID(),
                    promptId:'default'
                })
            });

			if (response.ok) {
				console.log('response',response);
				let stream = events(response);
				for await (let event of stream) {
					console.log('<<', event.data);
					try {
						const jsonData = JSON.parse(event.data);
					} catch (error) {
						sRes.addStreamRes(PUBLIC_AI_CONFIG, chatID, `${event.data} `);
					}
					
				}
			}
```

**Comment by JohnRSim** [comment]
![image](https://github.com/lukeed/fetch-event-stream/assets/364208/c536e249-8d90-4aa5-857c-d658ec03b7ef)

**Comment by JohnRSim** [comment]
Here is the response 
![image](https://github.com/lukeed/fetch-event-stream/assets/364208/1f3162dd-fd57-4498-a8a9-2c37327000ee)

**Comment by lukeed** [comment]
Ah, shoot. Sorry about this. Didn't realize that [`@@asyncIterator`](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream#browser_compatibility) for ReadableStreams was experimental & not-well supported in browsers.

I'll change that, either via #1 or through the linked commit (incoming). Do you mind testing it out locally for me, to verify it works for you?

**Comment by lukeed** [comment]
Fixed published in `0.1.4`

**Comment by JohnRSim** [comment]
@lukeed this is great thanks @lukeed tested with .5 works well.
